tests/dynamic_data/publications/timeseries/timeseries_test.py

```diff
@@ -112,6 +112,35 @@
         'expected_thumbnail': os.path.join(DIRECTORY, 'thumbnail_longname.png'),
         'wms_bbox': [1669480.0, 6580973.0, 1675352.0, 6586999.0],
     },
+    'longname_one_file': {
+        'params': {
+            'time_regex': r'[0-9]{8}',
+            'file_paths': [
+                os.path.join(DIRECTORY,
+                             'timeseries_tif/210_long_name_20221031_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.tiff'),
```

Comment:
Use sample_png_aux_rgba_opaque.png + *.aux.xml in ZIP file instead of current TIFF

Reply:
or sample_jpg_aux_rgba.jpg